libs/models/player.ts

```diff
@@ -0,0 +1,47 @@
+/** Interface for time-logs when players are typing */
+interface TimeLog {
+  /** Correctly typed length by player */
+  textLength: number;
+
+  /** Time when player typed the last character */
+  timestamp: number;
+}
+
+enum playerState {
```

Comment:
`enum PlayerState`

Reply:
Please use `PascalCase` for the enum names and value definitions.
Keep the enum values in `skewer-case`

Reply:
Fixed